composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "evilfreelancer/routeros-api-php",
+  "name": "proner/routeros-api-php",
```

Comment:
Same as in README.md

Reply:
Sorry, i changed vendor name for my fork and forgeted back changed before pull request